x-pack/plugins/security_solution/public/management/components/privileged_route/privileged_route.test.tsx

```diff
@@ -74,7 +73,6 @@ describe('PrivilegedRoute', () => {
       render();
 
       expect(renderResult.queryByTestId(componentTestId)).toBeNull();
-      expect(renderResult.queryByTestId(noPermissionsPageTestId)).toBeNull();
```

Comment:
I think there is no need to remove these assertions from lines 67 and 77. Independently from the feature flag, these tests ensure that neither the old nor the new access denied page is displayed. I tried locally, the tests should pass with these lines.

Reply:
updated